src/app/Plans/components/Wizard/HooksForm.tsx

```diff
@@ -184,15 +184,11 @@ const HooksForm: React.FunctionComponent<IHooksFormProps> = ({
           );
           toggleRemoveModal();
         }}
-        title="Remove hook"
+        title="Remove hook?"
```

Comment:
Let's add "permanently" here to emphasize the weight of this action. If this action isn't permanent though, we can remove it!

**Permanently remove hook?**
This pre-migration hook won't be executed when you run your migration plan.

or, for the "post" hook:

This post-migration hook won't be executed when you run your migration plan.